client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -15,33 +16,60 @@ import DocumentHead from 'components/data/document-head';
 import LostPasswordForm from 'account-recovery/lost-password-form';
 import ForgotUsernameForm from 'account-recovery/forgot-username-form';
 import ResetPasswordForm from 'account-recovery/reset-password-form';
+import ResetPasswordEmailForm from 'account-recovery/reset-password-email-form';
+import ResetPasswordSmsForm from 'account-recovery/reset-password-sms-form';
 import { ACCOUNT_RECOVERY_STEPS as STEPS } from 'account-recovery/constants';
 import {
 	isAccountRecoveryResetOptionsReady,
 	isAccountRecoveryUserDataReady,
+	getAccountRecoveryResetSelectedMethod,
 } from 'state/selectors';
 
 const getPageInfo = ( translate, step ) => {
+	const concatHeadTitle = ( parentTitle, childTitle ) => ( parentTitle + ' ‹ ' + childTitle );
+
 	const pageInfo = {
 		[ STEPS.LOST_PASSWORD ]: {
 			trackerTitle: 'Account Recovery > Lost Password',
-			documentHeadTitle: translate( 'Lost Password ‹ Account Recovery' ),
+			documentHeadTitle: concatHeadTitle( translate( 'Lost Password' ), translate( 'Account Recovery' ) ),
 		},
 		[ STEPS.FORGOT_USERNAME ]: {
 			trackerTitle: 'Account Recovery > Forgot Username',
-			documentHeadTitle: translate( 'Forgot Username ‹ Account Recovery' ),
+			documentHeadTitle: concatHeadTitle( translate( 'Forgot Username' ), translate( 'Account Recovery' ) ),
 		},
 		[ STEPS.RESET_PASSWORD ]: {
 			trackerTitle: 'Account Recovery > Reset Password',
-			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
+			documentHeadTitle: concatHeadTitle( translate( 'Reset Password' ), translate( 'Account Recovery' ) ),
+		},
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: concatHeadTitle( translate( 'Reset Password' ), translate( 'Email' ) ),
+		},
+		[ STEPS.RESET_PASSWORD_SMS ]: {
+			trackerTitle: 'Account Recovery > Reset Password Sms',
+			documentHeadTitle: concatHeadTitle( translate( 'Reset Password ' ), translate( 'SMS Message' ) ),
```

Comment:
Oops. I was thinking what went wrong and you beat me to it :D 
Updated in 920a0226dbfc55faca877507e2a744e487066811.